go/pkg/project/experiment.go

```diff
@@ -6,27 +6,27 @@ import (
 	"sort"
 	"time"
 
-	"github.com/replicate/replicate/go/pkg/config"
-	"github.com/replicate/replicate/go/pkg/console"
-	"github.com/replicate/replicate/go/pkg/hash"
-	"github.com/replicate/replicate/go/pkg/param"
-	"github.com/replicate/replicate/go/pkg/repository"
+	"github.com/replicate/keepsake/go/pkg/config"
+	"github.com/replicate/keepsake/go/pkg/console"
+	"github.com/replicate/keepsake/go/pkg/hash"
+	"github.com/replicate/keepsake/go/pkg/param"
+	"github.com/replicate/keepsake/go/pkg/repository"
 )
 
 // Experiment represents a training run
 type Experiment struct {
-	ID               string            `json:"id"`
-	Created          time.Time         `json:"created"`
-	Params           param.ValueMap    `json:"params"`
-	Host             string            `json:"host"`
-	User             string            `json:"user"`
-	Config           *config.Config    `json:"config"`
-	Command          string            `json:"command"`
-	Path             string            `json:"path"`
-	PythonVersion    string            `json:"python_version"`
-	PythonPackages   map[string]string `json:"python_packages"`
-	Checkpoints      []*Checkpoint     `json:"checkpoints"`
-	ReplicateVersion string            `json:"replicate_version"`
+	ID              string            `json:"id"`
+	Created         time.Time         `json:"created"`
+	Params          param.ValueMap    `json:"params"`
+	Host            string            `json:"host"`
+	User            string            `json:"user"`
+	Config          *config.Config    `json:"config"`
+	Command         string            `json:"command"`
+	Path            string            `json:"path"`
+	PythonVersion   string            `json:"python_version"`
+	PythonPackages  map[string]string `json:"python_packages"`
+	Checkpoints     []*Checkpoint     `json:"checkpoints"`
+	KeepsakeVersion string            `json:"keepsake_version"`
```

Comment:
Will `replicate_version` get clobbered if this is loaded and saved?

I was also thinking -- this needs to be read in a backwards compatible way, but I don't think `replicate_version` / `keepsake_version` is actually surfaced anywhere in UI, so I think it's safe to just migrate this and have any potential future usage of this data check for both.

Either way, we should probably keep `ReplicateVersion` in here as `omitempty` to remind us to check for it it in a future potential usage of this data.